ui/fisher/fisher.js

```diff
@@ -75,7 +75,7 @@ class Fisher {
         cast: NetRegexes.gameLog({
           code: '08c3',
           line:
-            '(?:[\\w\']\\.?)(?:[\\w\'\\s]+\\.?)? cast(?:s?) (?:your|his|her) line (?:on|in|at) (?:the )?(?<place>[\\w\\s\'&()]+)\\..*?',
+            '(?:[\\w\']\\.?)(?:[\\w\'\\s]+\\.?)? cast(?:s?) (?:your|his|her) line (?:on|in|at) (?:the )?(?<place>.+)\\..*?',
```

Comment:
I think this is ok because no fishing locations (currently) have a `.` in them, and so the `\\.` will always catch the end of the fishing place.